src/handlers/check_commits.rs

```diff
@@ -7,6 +7,9 @@ use crate::{
     github::{Event, IssuesAction, IssuesEvent, ReportedContentClassifiers},
 };
 
+#[cfg(test)]
+use crate::github::GithubCommit;
```

Comment:
You can also do the import locally in the `dummy_commit_from_body` function and spell out the full path to the return type to avoid the cfg. I would perhaps move these test helpers functions to the dedicated `tests` module, but we don't need to do that in this PR.

Reply:
I can't just import it in `dummy_commit_from_body` since it's used as the return type of the function.

Reply:
Yeah as I saw, you could fully spell out the absolute path in the return type, but it doesn't matter.